sklearn/ensemble/gradient_boosting.py

```diff
@@ -1836,3 +1843,353 @@ def apply(self, X):
         leaves = super(GradientBoostingRegressor, self).apply(X)
         leaves = leaves.reshape(X.shape[0], self.estimators_.shape[0])
         return leaves
+
+
+class GradientBoostingClassifierCV(BaseEstimator):
+    """ Gradient Boosting Classification with Cross Validation
+
+    This class keeps adding estimators to the underlying GB estimator till
```

Comment:
@arjoly Could you please explain what you mean?

How is that different from doing something like

gscv = GridSearchCV(GradientBoostingRegressor(), {n_estimators: [1, 2, .. n]})
gscv.fit(X, y)